src/main/java/info/sjd/controller/CartController.java

```diff
@@ -65,10 +65,9 @@
 
     @PostMapping(path = "update_status")
     public ResponseEntity<Cart> updateStatus(@RequestParam Integer idParam,@RequestParam Integer closedParam){
-        Cart cart = cartService.updateStatus(idParam, closedParam);
-        if (cart != null){
-            return new ResponseEntity<>(cartService.updateStatus(idParam,closedParam), HttpStatus.OK);
+        if (updateStatus(idParam, closedParam) != null){
```

Comment:
What is it?! You call the same method in this method recursively? Why? 
You should get results from service and check it here to send correct response.

Reply:
CarConrollert updateStatus done, and OrderConrollert updateAmount done